internal/store/driver.go

```diff
@@ -711,8 +711,8 @@ func (d *Driver) calculateTunnels() map[tunnelKey]ingressv1alpha1.Tunnel {
 						Name:       ingress.Name,
 						UID:        ingress.UID,
 					})
-					slices.SortStableFunc(tunnel.OwnerReferences, func(i, j metav1.OwnerReference) bool {
-						return i.UID < j.UID
+					slices.SortStableFunc(tunnel.OwnerReferences, func(i, j metav1.OwnerReference) int {
+						return strings.Compare(string(i.UID), string(j.UID))
```

Comment:
our of curiosity - does [cmp.Compare](https://pkg.go.dev/cmp#Compare) work?

Reply:
fixed here: https://github.com/ngrok/kubernetes-ingress-controller/pull/328/commits/2133735d809fee5827a15438a806092d5e5545b6